File: src/main/java/recipeapplication/model/Step.java
package recipeapplication.model;

import javax.persistence.*;

@Entity
@Table(name = "steps")
public class Step
{
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String description;

    public Step() {}

    public Step(String description)
    {
        this.description = description;
    }

    public void setDescription(String description)
    {
        this.description = description;
    }

    public String getDescription()
    {
        return description;
    }

    public void setId(Long id)
    {
        this.id = id;
    }

    public Long getId()
    {
        return id;
    }
}

File: src/main/kotlin/recipeapplication/model/Step.kt
package recipeapplication.model

import javax.persistence.*

@Entity
@Table(name = "steps")
class Step {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null
    lateinit var description: String

    constructor()
    constructor(description: String) {
        this.description = description
    }
}